app/design/adminhtml/default/default/template/ebizmarts/mailchimp/system/config/form/field/array_dropdown.phtml

```diff
@@ -45,16 +45,16 @@ $colspan = $colspan > 1 ? 'colspan="' . $colspan . '"' : '';
                     </span>
                 </button>
             </td>
-            <td <?php echo $this->quoteEscape($colspan) ?>>
+            <td <?php echo $this->gethelper()->mcEscapeQuote($colspan) ?>>
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.